vueui/README.md

```diff
@@ -0,0 +1,180 @@
+# vueui - Vue.js based UI framework for SS13
+This UI framework is mostly composed of four main parts:
+ - Main JS file and supporting HTML code
+ - UI DM datum, responsible for linking opened ui window with code
+ - Object specific code, that generates data for uis, and handles actions.
+ - Sub System responsible for keeping track of all uis and making sure they tick.
+ 
+## How to use it?
+### Step 1: Open ui
+First we have to create a way to open ui, for example, a proc that's called when we want to open ui:
+```DM
+/datum/mydatum/proc/open_ui()
+    var/datum/vueui/ui = SSvueui.get_open_ui(usr, src)
+    if (!ui)
+        ui = new(usr, src, "uiname", 300, 300, "Title of ui")
+    ui.open()
```

Comment:
We open the UI that `get_open_ui()` returns? `get_open_ui()` can return a non-open UI?

Reply:
Well, in rare case, ui can get closed without server noticing it, this also makes sure if user clicks on machine again, it updates already open ui.